pyserini/resources/index-metadata/faiss.mrtydi-v1.1.20220524.7b099d5.mdpr-tied-pft-msmarco-ft-all.README.md

```diff
@@ -0,0 +1,54 @@
+# mrtydi-v1.1-arabic (pre--fine-tuned on MS MARCO then fine-tuned on all Mr. TyDi languages)
+
+Faiss flat index for Mr.TyDi v1.1 (Arabic), using mDPR fine-tuned on NQ.
+
+This index was generated on 2022/05/24 at commit [7b099d5](https://github.com/crystina-z/pyserini/commit/7b099d534901d1f0161982605cd40d039ddb701d) on `basilisk` with the following command:
+
+```bash
+lang=arabic # any language in Mr. TyDi
+
+tarfn=mrtydi-v1.1-$lang.tar.gz
+corpus=mrtydi-v1.1-$lang/collection/docs.jsonl
+index_dir=mrtydi-mdpr-dindex-msmarco/$lang
+
+wget https://git.uwaterloo.ca/jimmylin/mr.tydi/-/raw/master/data/$tarfn
+tar –xvf $tarfn
+gzip -cvf $corpus.gz > $corpus
+
+shard_num=1
+encoder=castorini/mdpr-tied-pft-msmarco-ft-all
+
+for shard_id in $(seq 0 `$shard_num - 1`) ; do
+    index_dir=mdpr-dindex/$lang-$shard_id
+    mkdir -p $index_dir
+    python -m pyserini.encode   input   --corpus $corpus \
+                                        --fields title text \
+                                        --delimiter "\n\n" \
```

Comment:
why it is `\n\n` here?

Reply:
oh it's encoding corpus in mrtydi format, which looks like "{title}\n\{text}" under contents